G/git_crypt/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "git_crypt"
+version = v"0.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/AGWA/git-crypt/archive/0.6.0.tar.gz" =>
+    "777c0c7aadbbc758b69aff1339ca61697011ef7b92f1d1ee9518a8ee7702bb78",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd git-crypt-0.6.0/
+export CXXFLAGS=-I${prefix}/include
```

Comment:
`CXXFLAGS` are the flags for the C++ compiler.  Here you probably want to use `CPPFLAGS`, the flags for the C preprocessor.  It may work if the program is written in C++.